### ISSUE TITLE
bad accuracy on test data

### ISSUE THREAD

**Issue by Caiji-123** [plkmo/BERT-Relation-Extraction#31]
Hi, I find your accuracy is calculated on training data, but generally we evaluate the model on test data. I get the  accuracy from test data and the result is not that good, is there anything wrong?

**Comment by Caiji-123** [comment]
the acc on test I got is 0.588
![image](https://user-images.githubusercontent.com/66150905/101600102-cbb13c80-3a35-11eb-9609-fc43e56daefe.png)
epoch 38

**Comment by plkmo** [comment]
Hi there, for semeval task the model is evaluated on test data. However for pre-training, there is no test data as the pre-training is self-supervised. You were probably looking at the code for pre-training.

Also the accuracy is evaluated on a multi-class > 2 dataset, so the baseline accuracy is not 0.5. You can use F1 for a better comparison

**Comment by Caiji-123** [comment]
已收到您的邮件
